pkg/webui/account/views/front/index.js

```diff
@@ -43,6 +47,11 @@ const FrontView = () => {
             <Route path="/validate" component={Validate} />
             <Route path="/code" component={Code} />
             <Redirect exact from="/" to="/login" />
+            {authRoutes.map(({ path, exact }) => (
```

Comment:
Currently, there is only one route in `authRouter` which is for `/`. When the user navigates to `/oauth` these lines are never reached because of L49

Reply:
That's true. There will be more routes added with the next PRs which will make this setup purposeful. It's fine that `/` is handled by L49, since we don't need a redirect query parameter for it.